pyproject.toml

```diff
@@ -7,24 +7,24 @@ include = ["src/*"]
 
 [tool.poetry-dynamic-versioning]
 #set to false.  poetry-dynamic-versioning only needed for CI versioning.
-enable = false
+enable = true
 vcs = "git"
 style = "pep440"
 
 [tool.poetry-dynamic-versioning.substitution]
 #poetry-dynamic-versioning has auto subsitution for __init__.py files,
 #but not for more than 1 directory out of root.
-files = ["src/epcpm/__init__.py"]
+#files = ["src/epcpm/__init__.py"]
```

Comment:
I checked the `__init__.py` file in the build artifact and the `__version__` was still set to `0.0.0`. This row is likely necessary.

Reply:
Resolved.